ObjObsSAP.tex

```diff
@@ -256,7 +255,7 @@ \subsection{\{query\} resource} \label{sec:query}
 names refer to those defined in the ObsCore Data Model .
 
 Some of the parameters proposed in this standard are not described in
-the ObsCore Data Model document , such as; min\_vis, max\_vis,
+the ObsCore Data Model document , such as; min\_obs, max\_obs,
```

Comment:
`max_obs` is mentioned here, but is never described in this document, on the contrary to `min_obs`.
Is it normal?

Reply:
Thanks Gregory, I think all the points mentioned are relevant. I would let Aitor to edit directly the doc (I said to him that if he has some problem pushing from command line, small changes could be done directly in the portal). In case he still face problem pushing changes to this pull request, I could suppport

Reply:
If I remember correctly, we discussed time ago about to add max_obs as input. But then we decided to move it an optional output parameter.
I have added the description in the "Standard output fields" section.